source/codegen/metadata/nidmm/functions.py

```diff
@@ -26,155 +26,6 @@
         ],
         'returns': 'ViStatus'
     },
-    "CalAdjustGain": {
```

Comment:
@jonlugoatx would you prefer removing these metadata or marking them with `codegen: no`? I see we have few APIs marked with `codegen: no` elsewhere.

I guess advantage of retaining metadata would be 
1. Less changes from base metadata. So it would be easier to merge if we ever want to sync from base metadata or regenerate the metadata tile.
2. In case someone wants to use these APIs, it would be easier for them to regenerate. Not sure if we want this.

Removing the metadata on the other hand, would just keep it cleaner.

This PR doesn't have any attributes, but any external calibration related attributes will need to be removed from metadata since we don't have support for `codegen: no` for attributes.

FYI @Akshata440 for [this PR](https://github.com/doshirohan/grpc-device/pull/123).

Reply:
Removing them is okay. When we take on the work to integrate with hapigen hiding the work to remove vs hide them would be the same.